### ISSUE TITLE
equivalent function SQL Server's TRANSLATE() to in Presto

### ISSUE THREAD

**Issue by laritsuda** [prestosql/presto#3313]
I need to translate the special characters (in Portuguese language) of a string column into 'raw' characters, using Amazon Presto. In SQL Server, I could do that using the TRANSLATE function:

``` 
select
city_name,
translate(city_name,'áéíóúÁÉÍÓÚäëïöüÄËÏÖÜâêîôûÂÊÎÔÛãẽĩõũÃẼĨÕŨ','aeiouAEIOUaeiouAEIOUaeiouAEIOUaeiouAEIOU') as cityname_raw
FROM cities
```

That would return exactly what I need:

city_name
---------
Goiânia	 
São Paulo	
Palhoça  
Várzea Paulista

city_name_raw
-------------
Goiania	 
Sao Paulo	
Palhoca  
Varzea Paulista

However, I didn't find an equivalent function in Presto. Could it be developed?
I realize there is a workaround using several nested replaces, but translate() would be better.

**Comment by tooptoop4** [comment]
https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFTranslate.java

**Comment by ebyhr** [comment]
Going to close this issue since the function was implemented in #4080
